library/psa_crypto_aead.c

```diff
@@ -52,7 +52,7 @@ static psa_status_t psa_aead_setup(
 
     key_bits = attributes->core.bits;
 
-#if defined (MBEDTLS_PSA_CRYPTO_ACCEL_DRV_C)
+#if defined (MBEDTLS_PSA_CRYPTO_ACCEL_DRV_C) && (defined(MBEDTLS_GCM_ALT) || defined(MBEDTLS_CCM_ALT) || defined(MBEDTLS_CIPHER_ALT))
```

Comment:
This condition is a bit confusing IMO. 
The way it is now, it will return true if "MBEDTLS_PSA_CRYPTO_ACCEL_DRV_C" is defined AND MBEDTLS_GCM_ALT. Or it will return true if either of MBEDTLS_CCM_ALT OR MBEDTLS_CIPHER_ALT are defined. 
If that is the intention then please add parentheses around the first two.

Reply:
NVM. I didnt see a parentheses.